experimental/orchestrator/csharp_dotnetcore/02.todo-bot-with-adaptive-dialog/README.md

```diff
@@ -21,7 +21,7 @@ This sample **requires** prerequisites in order to run.
     ```
     - Install bf orchestrator
     ```bash
-    > bf plugins:install @microsoft/bf-orchestrator-cli@beta
+    > bf plugins:install @microsoft/bf-orchestrator-cli@latest
```

Comment:
Not needed, we'll tag latest deployment with beta tag.

Reply:
reverted